WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -345,7 +345,16 @@ import WordPressComAnalytics
     func configureCell(cell: BasePageListCell, atIndexPath indexPath: NSIndexPath) {
         cell.accessoryType = .None
         cell.selectionStyle = .None
-        cell.delegate = self
+        
+        if cell.reuseIdentifier == self.dynamicType.pageCellIdentifier {
+            cell.onAction = { [weak self] (cell: BasePageListCell, button: UIButton, provider: WPPostContentViewProvider) in
+                self?.handleMenuAction(fromCell: cell, fromButton: button, forProvider: provider)
+            }
+        } else if cell.reuseIdentifier == self.dynamicType.restorePageCellIdentifier {
+            cell.onAction = { [weak self] (cell: BasePageListCell, button: UIButton, provider: WPPostContentViewProvider) in
```

Comment:
In fact, because we don't use the `button` here, we could do:

```
cell.onAction = { [weak self] cell, _, provider in
    self?.handleRestoreAction(fromCell: cell, forProvider: provider)
}
```